readme-sync/v0/senseml-reference/8000 - concepts/1800 - jsonlogic.md

```diff
@@ -236,6 +236,8 @@ Returns a JSON object that is an array of key/value pairs. You can nest object o
 }
 ```
 
+### Examples
```

Comment:
added an Examples header here as a reminder to include something, but didn't know what would be best to link to

Reply:
hi! I removed the `object` example originally because we'd decided that in most cases, `eachKey` would be better as an alternative. To my mind it's not high priority to add an example back in, unless you're encountering more use cases for it.